sympy/core/assumptions.py

```diff
@@ -196,13 +196,29 @@ class StdFactKB(FactKB):
     """
     rules = _assume_rules
 
-    def __init__(self, facts=None):
+    def __init__(self, facts=None, extra_facts=None):
```

Comment:
Can you explain this change? I don't really understand what is going on. Is this just to capture any assumptions that are added after the ones passed in by the user?